roles/opensrp/defaults/main.yml

```diff
@@ -46,6 +46,7 @@ allowed_couchdb_hosts: []
 mysql_backups_to_s3: false
 tomcat_user: "tomcat7"
 tomcat_group: "tomcat7"
+opensrp_tomcat_version: 7
```

Comment:
If you update the default OpenSRP version to `v2.0.0` and `opensrp_java_packages` to `openjdk-11-jre`, it might be a good idea to also set this to `9`.

Reply:
Okay. Let me change